org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/tracking/Trackable.java

```diff
@@ -62,8 +62,26 @@
   // empty if none
   String getAssignee();
 
+  /**
+   * Can be overriden by server side issue in connected mode
+   */
   String getSeverity();
 
+  /**
+   * Original severity reported by the analyzer
+   */
+  String getRawSeverity();
+
+  /**
+   * Can be overriden by server side issue in connected mode
+   */
+  String getType();
+
+  /**
+   * Original type reported by the analyzer
+   */
+  String getRawType();
```

Comment:
I think "raw" usually means "unprocessed", not ready for use. I would have used `getOriginalType` and `getOriginalSeverity`, but it's up to you.